### PR TITLE
[webserver] Make BE webserver more pretty

### PR DESCRIPTION
**Pull request by acelyc111**
Add some CSS and js files, and use boost-table framework to make BE's website more pretty

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -212,6 +212,7 @@ if [ ${BUILD_BE} -eq 1 ]; then
     cp -r -p ${DORIS_HOME}/be/output/bin/* ${DORIS_OUTPUT}/be/bin/
     cp -r -p ${DORIS_HOME}/be/output/conf/* ${DORIS_OUTPUT}/be/conf/
     cp -r -p ${DORIS_HOME}/be/output/lib/* ${DORIS_OUTPUT}/be/lib/
+    cp -r -p ${DORIS_HOME}/be/output/www/* ${DORIS_OUTPUT}/be/www/
```
Comment (morningman) [inline build.sh:215]: You need to add `www/` to the above `install -d` command.

File: webroot/be/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+-->
+<html>
+<body>
+<h1>Doris Webserver</h1>
```
Comment (morningman) [inline webroot/be/index.html:22]: How about:
`Doris Backend`
?

**Review by morningman — CHANGES_REQUESTED** [review]

**Comment by acelyc111** [comment]
Now BE's website looks like:
image

**Comment by acelyc111** [comment]
Also a step for https://github.com/apache/incubator-doris/issues/3911

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**